unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/PrivateChatWindow/PrivateChatWindowController.cs

```diff
@@ -299,6 +311,8 @@ public void ActivatePreviewMode()
         OnPreviewModeChanged?.Invoke(true);
     }
 
+    public void SetLoadingMessagesActive(bool isActive) { View?.SetLoadingMessagesActive(isActive); }
```

Comment:
this method should not be public if not needed.. in case of tests you should trigger events to assert the flow

Reply:
As I explain in the previous comment, this `SetLoadingMessagesActive()` function is being currently called by the `TaskbarHUDController`.

Reply:
With the refactor mentioned [above](https://github.com/decentraland/unity-renderer/pull/2484/commits/3353be0a20852adf41295c15a29eabd0f1cf4959), I'm not longer need to have it as public 👍 So it's already fixed.